src/components/TimeCardPage/SubmitCard.tsx

```diff
@@ -8,39 +8,57 @@ import Button from 'react-bootstrap/Button';
 
 export default function SubmitCard(props) {
 
-    const [submitted, setSubmitted] = useState(false); 
-    const [submitDate, setSubmitDate] = useState(null); 
+    const CardState = {
+        Rejected: "Rejected",
+        InReviewEmployer: "In Review - Employer",
+        InReviewBreaktime: "In Review - Breaktime",
+        Completed: "Completed"
+    }
+
+    const [submitted, setSubmitted] = useState(false);
+    const [submitDate, setSubmitDate] = useState(null);
+    const [state, setState] = useState(CardState.Rejected);
+    const [rejected, setRejected] = useState(false);
 
     useEffect(() => {
         //TODO - API Call to determine if the table has been submitted or not.
-        //Will set submitted? here and also submitDate if it was submitted to grab the date 
-        
-    },[])
+        //Will set submitted? here and also submitDate if it was submitted to grab the date     
+    }, [props.comment])
 
     const submitAction = () => {
-        setSubmitted(!submitted); 
-        const currentTime = new Date(); 
-        setSubmitDate(currentTime.toString()); 
+        setSubmitted(!submitted);
+        const currentTime = new Date();
+        setSubmitDate(currentTime.toString());
+        if (state === CardState.Rejected) {
+            setState(CardState.InReviewEmployer);
+        }
+        else {
+            setState(CardState.Rejected);
+        }
     }
 
     return (
-        <div className="col-md-2" style={{ display: "flex", justifyContent: 'flex-end'}}>
-            <Card 
-                bg={submitted ? 'success' : 'secondary'}
+        <div className="col-md-2" style={{ display: "flex", justifyContent: 'flex-end' }}>
```

Comment:
Can this div be replaced with Chakra's [Box](https://chakra-ui.com/docs/components/box)? Prefer for consistent styling